x-pack/plugins/security_solution/public/management/components/endpoint_responder/console_exit_modal_info.tsx

```diff
@@ -8,20 +8,26 @@
 import { i18n } from '@kbn/i18n';
 import React, { memo } from 'react';
 import { FormattedMessage } from '@kbn/i18n-react';
-import { EuiText } from '@elastic/eui';
+import { EuiText, EuiToolTip } from '@elastic/eui';
 
 import { CONSOLE_EXIT_MODAL_INFO } from '../console/components/console_manager/translations';
 
 const actionLogLink = i18n.translate('xpack.securitySolution.consolePageOverlay.exitModal.link', {
   defaultMessage: 'Action log',
 });
 
+const getTruncateHostName = (name: string): string => {
+  if (name.length <= 12) {
+    return name;
+  }
+  const split = [name.slice(0, 8), name.slice(-4)];
+  return split.join('..');
+};
+
 export const HostNameText = ({ hostName }: { hostName: string }) => (
-  <EuiText size="s" style={{ maxWidth: 100, display: 'inline-flex' }}>
-    <span className="eui-textTruncate">
-      <strong>{hostName}</strong>
-    </span>
-  </EuiText>
+  <EuiToolTip content={hostName}>
```

Comment:
Optional suggestion:

Remove `getTruncatedHostName()` and move the logic here and place it in a `useMemo()` so that it is more efficient (no need to do the split/join on every re-render): 

```typescript
const displayValue = useMemo(() => {
  if (name.length <= 12) {
    return name;
  }
  const split = [name.slice(0, 8), name.slice(-4)];
  return split.join('..');
}, []);

```

Reply:
811c76f970115733013c906425d3ce406d83f051